Extra note on summary from new coverage (#1979)
codacy/docs: docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

| Feature                                          | Status      | Notes                                                                                                                                                       |
|--------------------------------------------------|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [Git provider status checks](#status-checks)     | Live (beta) | The new Coverage engine now sends coverage data to your Git provider. This data is marked \[beta\] and is shown alongside the data from the current engine. |

> | [GitHub coverage summaries](#coverage-summaries) | Live        | Summaries are now sent from the new Coverage engine. If you haven't done so yet, please review and update your app permissions as mentioned above.          |

Reviewer: ```suggestion
| [GitHub coverage summaries](#coverage-summaries) | Live        | The new Coverage engine now sends coverage summaries to GitHub. If you haven't done so yet, please review and update your app permissions as mentioned above.          |
```
